src/app/components/team/SwitchTeamsComponent.js

```diff
@@ -157,9 +159,14 @@ class SwitchTeamsComponent extends Component {
                   }
                 />
                 <ListItemSecondaryAction>
-                  <IconButton edge="end" onClick={this.setCurrentTeam.bind(this, team, currentUser)}>
-                    <KeyboardArrowRight />
-                  </IconButton>
+                  <ButtonMain
+                    iconCenter={<KeyboardArrowRight />}
+                    theme="text"
+                    variant="text"
+                    size="default"
+                    edge="end"
```

Comment:
is edge a ButtonMain prop?

Reply:
good catch @amoedoamorim 
Thanks for the review
I missed this when converting. I will remove once (hopefully) the integrations test pass

Reply:
ok removed, integration tests didnt pass for other reasons, but this was resolved